File: poo/2. User Registration at a Gym JS/premium.java
public class Premium extends CalculateTotal {
    private int hour;
    private int coach;
    private int humidArea;
    private int value;
    private int calculate;

    public Premium(int hour) {
        super("Essential", "Basico"); 
        this.hour = hour;
        this.coach = 50000;
        this.humidArea = 70000;
        this.value = 40000;
    }

    public int getHour() {
        return hour;
    }

    public int getCoach() {
        return coach;
    }

    public int getHumidArea() {
        return humidArea;
    }

    public int getValue() {
        return value;
    }

    @Override
    public double calculate() {
        calculate = hour * value + coach + humidArea;
        return calculate;
    }
}

File: poo/2. User Registration at a Gym/premium.php
<?php
    class Premium extends CalculateTotal {
        private int $hour;
        private int $coach;
        private int $humidArea;
        private int $value;
        private int $calculate;

        public function __construct(int $hour) {
            parent::__construct("Essential", "Basico");
            $this->hour = $hour;
            $this->coach = 50000;
            $this->humidArea = 70000;
            $this->value = 40000;
        }

        public function getHour(): int {
            return $this->hour;
        }
        public function getCoach(): int {
            return $this->coach;
        }
        public function getHumidArea(): int {
            return $this->humidArea;
        }
        public function getValue(): int {
            return $this->value;
        }

        public function calculate(): float {
            $this->calculate = $this->hour * $this->value + $this->coach + $this->humidArea;
            return $this->calculate;
        }
    }
?>